app/main.py

```diff
@@ -1,2 +1,68 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity: int = 8
+        self.threshold: float = 0.66
+        self.size: int = 0
+        self.hash_table: list = [None] * 8
```

Comment:
Annotations for the attributes are not really necessary here. Type hints are already clear from the assigned values. You can skip annotations in such cases to write less code